src/main/java/org/sagebionetworks/web/client/widget/profile/UserProfileEditorWidgetViewImpl.java

```diff
@@ -405,6 +418,25 @@ public void setOwnerId(String userId) {
     userProfileLinksReactComponentContainer.render(profileLinksComponent);
   }
 
+  public void setAccountTypeVisibility(Long userId) {
+    int mask = IS_CERTIFIED | IS_VERIFIED;
+    jsClient.getUserBundle(
+      userId,
+      mask,
+      new AsyncCallback<UserBundle>() {
+        @Override
+        public void onSuccess(UserBundle bundle) {
+          boolean showAccountType =
+            bundle.getIsCertified() | bundle.getIsVerified();
+          accountTypeContainer.setVisible(showAccountType);
+        }
+
+        @Override
+        public void onFailure(Throwable caught) {}
```

Comment:
If an error occurs when getting the UserBundle, the `AccountLevelBadges` component will display an error. Since the `accountTypeContainer` defaults to visible, I think that the `AccountLevelBadges` component error will be visible, so we may not need to handle the failure here. Happy to add in handling though, if that'd be preferred!